src/main/java/backend/IssueMetadata.java

```diff
@@ -54,7 +55,7 @@ public IssueMetadata invalidate() {
      * is filled in later.
      */
     public static IssueMetadata intermediate(List<TurboIssueEvent> events, List<Comment> comments,
-                                             String eventsETag, String commentsETag) {
+            String eventsETag, String commentsETag) {
```

Comment:
This is allowed by checkstyle. However, the continuous indentation for lambda expressions should be 4 spaces less than normal line-wrapping indentation. Initially i tot this was a bug, but after reading https://github.com/checkstyle/checkstyle/issues/281#issuecomment-144733712 it seems that it is a design by purpose.